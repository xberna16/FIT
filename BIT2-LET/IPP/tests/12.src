.IPPcode18
DEFVAR GF@A
DEFVAR GF@B
DEFVAR GF@Vysledek
MOVE GF@A string@AAA
MOVE GF@B string@AAAB
WRITE string@A=AAA_B=AAAB
LT GF@Vysledek GF@A GF@B
WRITE string@LT
WRITE GF@Vysledek
GT GF@Vysledek GF@A GF@B
WRITE string@GT
WRITE GF@Vysledek
EQ GF@Vysledek GF@A GF@B
WRITE string@EQ
WRITE GF@Vysledek
