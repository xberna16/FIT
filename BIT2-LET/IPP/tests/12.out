A=AAA_B=AAAB
LT
true
GT
false
EQ
false
